### DISCUSSION TITLE
Why H.265/H.266/H.267 cannot beat H.264 at 1080p archival-grade encoding?

### DISCUSSION THREAD

**Discussion by birdie-github** [fraunhoferhhi/vvenc#495] [General]
I will put it bluntly: I don't care about 4K/8K encoded using relatively low bitrate settings. Not my use case by a long shot.

Why can't new codecs beat ages old H.264 at 1080p **archival-grade encoding**, that is when visually (nearly completely lossless) lossless encoding is at stake?

It's not just [my sentiment](https://forum.doom9.org/showthread.php?p=2012016#post2012016)/research, most people agree with that.

E.g.

> Yes, x265 can achieve archival-grade quality with the right settings (no-sao and the like). But I share the sentiment that all these modern codecs are a let-down, it being a struggle to achieve transparency with them at reasonable bitrates. I sometimes wonder where all those CPU cycles go. It's as if H.264 hit critical mass of compression technology.

> We have shifted to we need higher / best quality at minimal bitrate, instead of having bare minimum / good enough quality at lowest possible bitrate. The two are very different.

In terms of computational complexity vvenc is probably 100 to 1000 times more expensive than x264 but for grainy 1080p sources it often needs higher bitrate to preserve all the detail. That sounds almost crazy.

**Comment by birdie-github** [comment]
@adamjw24
@K-os

**Reply by adamjw24** [reply]
For near-lossless you'd need a near-lossless tune, removing some tools that reduce the granularity of reisduum (i.e. secondary transform, residual blocks larger than 32x32 or 16x16 for non DCTII). H.264 does not have many of those, so doing a normal encode with a full toolset seems to work alright.

I guess a different cost modelling might be required as well, if you really want near-lossless. Here probably x264 does a better job and adapts automatically.

VVC and HEVC are most of all distribution codecs. For archiving there are better codecs. I guess you'd also want an all-intra codec to protect agaist data loss.

From a more theoretical point of view, the more towards lossless you go the less is there to gain. H.264 has a lossless bitrate savings of around 2x, VVC maybe 2.2x - depending on the content. Its because only real redundancies can be exploited, not irrelevances, and there is way more potential to expoit the latter.

So, its not your use-case, why not just go for codecs more suited to what you want to do, like pro-res, all-intra avc or jpeg-xl/xs (always confuse those...)?


PS: judging video quality from still frames as often discussed in the issues here is just bad practice

**Reply by birdie-github** [reply]
I wonder how it's possible to use ~100 more computing power at the same bitrate and achieve a much softer-looking image than I get with x264. Regardless of stills, I'm looking for fine details. That's what archival-grade encoding is, and they're just not there with vvenc.

I can email you all the details and settings privately so you can see for yourself.
